packages/@uppy/core/src/index.js

```diff
@@ -472,6 +472,13 @@ class Uppy {
 
     const fileID = generateFileID(file)
 
+    if (files[fileID]) {
+      this.info(`Not adding duplicate file '${fileName}', it already exists`, 'error', 5000)
+      this.log(`Not adding duplicate file '${fileName}', it already exists`)
```

Comment:
Is this causing multiple log statements? If so, is it intended?

Reply:
Info is shown to the user, why log is logged to the console, but I’ve merged the id into the previous log statement!